addons/mail/static/src/web_editor_emoji_command/web_editor_emoji_command.js

```diff
@@ -0,0 +1,58 @@
+/** @odoo-module */
+
+import { _t } from 'web.core';
+import Wysiwyg from 'web_editor.wysiwyg';
+import { getRangePosition } from '@web_editor/js/editor/odoo-editor/src/utils/utils';
+
+Wysiwyg.include({
+    async willStart() {
+        await this._super();
+        this.messaging = await owl.Component.env.services.messaging.get();
```

Comment:
This `await` will delay the loading of all editors until messaging is initialized, I don't think it is desirable.

Code should be done in a non-blocking `then` ideally.